docs/proposals/timeseries-partitioning-in-compactor.md

```diff
@@ -122,6 +122,37 @@ Now that the planner has produced a compaction plan for the T1-T3 compaction gro
 ![Grouping](/images/proposals/timeseries-partitioning-in-compactor-compact.png)
 T1 - Partition 1-2 was created with hash % 2 == 0, and in order to avoid having duplication information in blocks produced by compaction group 3-8, compaction group 5-8, and compaction group 7-8, we need apply the filter the `%8 == 0` hash, as that’s the hash of the highest partition count.
 
+### Compaction Workflow
+
+1. Cortex Compactor initializes grouper and planner. Then, call Thanos Compactor to compact blocks.
```

Comment:
Would be a good idea to make it clear that the group and planner are passed to Thanos' compactor.